pkg/ttl/ttlworker/job_manager.go

```diff
@@ -286,6 +286,9 @@ func (m *JobManager) jobLoop() error {
 }
 
 func (m *JobManager) onTimerTick(se session.Session, rt *ttlTimerRuntime, syncer *TTLTimersSyncer, now time.Time) {
+	if !variable.EnableTTLJob.Load() {
```

Comment:
This line will make some logic changes. How about just stopping ttl manager in the tests? See:

https://github.com/pingcap/tidb/blob/6aef624c8d4fc6e0301c4ef2f527f3b8a056edd7/pkg/ttl/ttlworker/job_manager_integration_test.go#L823-L828